drivers/pds/lib/environment.go

```diff
@@ -0,0 +1,44 @@
+package lib
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/onsi/gomega"
+)
+
+const (
+	envControlPlaneURL    = "CONTROL_PLANE_URL"
+	envPDSTestAccountName = "TEST_ACCOUNT_NAME"
+	envTargetKubeconfig   = "TARGET_KUBECONFIG"
+	envUsername           = "PDS_USERNAME"
+	envPassword           = "PDS_PASSWORD"
+	envPDSClientSecret    = "PDS_CLIENT_SECRET"
+	envPDSClientID        = "PDS_CLIENT_ID"
+	envPDSISSUERURL       = "PDS_ISSUER_URL"
+	envClusterType        = "CLUSTER_TYPE"
+	envTargetClusterName  = "TARGET_CLUSTER_NAME"
+)
+
+// GetAndExpectStringEnvVar parses a string from env variable.
+func GetAndExpectStringEnvVar(varName string) string {
+	varValue := os.Getenv(varName)
+	gomega.Expect(varValue).NotTo(gomega.BeEmpty(), "ENV "+varName+" is not set")
```

Comment:
rather than expect retrun the errors and handle them in tests

Reply:
done